### ISSUE TITLE
Spruce trees don't cut down all the way, if you cut two at the sametime.

### ISSUE THREAD

**Issue by Fonzee9** [slipcor/TreeAssist#44]
![2020-04-10_17 25 32](https://user-images.githubusercontent.com/21323258/79024246-5d3b7800-7b50-11ea-9c57-6c4ecb1e9d7b.png)

Only happens with spruce trees, all other ones do work as intended when cutting two at the same time.

**Comment by Fonzee9** [comment]
Oh there is also maybe another glitch if you accidentally break the sapplings while the tree is being tree fellered it stops the tree feller.

**Comment by slipcor** [comment]
Yes. This was in there from back in the day when there were only 1-block stems and 4-block stems. I am removing this safeguard in the next version that will also address your other issue.

I do not think that breaking the replaced sapling stops the tree feller. afaik it can not be stopped unless you replace a block of the stem with something that would indicate it is player placed like a cobble block ^^

I will try to investigate this, but I am in the process of planning a big rewrite anyway for 1.16 - things will change big time and I will rewrite the tree determining process, as I have to due to the new nether structures that are SO different that it is time to rethink what a tree is and what it can have.

I will upload a test build on issue #43 and if you give me the go-ahead I will release this and freeze the code unless some breaking bug appears, until the 1.16 rewrite is done.

**Comment by Fonzee9** [comment]
Says it still happens, but now it's more stable now:

![Capture](https://user-images.githubusercontent.com/21323258/79050762-2e79dc00-7bfa-11ea-8d26-4eeb09cf348f.PNG)

**Comment by slipcor** [comment]
Okay! I hope you can hang in there while I basically rewrite the tree calculation, including support for tree farms with stems right next to each other in a long line, I assume these go a bit further in the direction but should be supported because they are clearly distinguishable from a player structure ^^

**Comment by slipcor** [comment]
Not to put pressure on people that might be reading this, but testing has started. Thanks for your patience, I strongly assume that version 7 will be released soon :D
